RELEASE_NOTES.md

```diff
@@ -0,0 +1,8 @@
+# Paloma v1.0.0 Release Notes
+
+The initial release of Paloma!
```

Comment:
@taariq / @verabehr feel free to add anything else here you think is relevant to the release or launch.